### DISCUSSION TITLE
apisix配合 oss 部署前端问题

### DISCUSSION THREAD

**Discussion by 88fantasy** [apache/apisix#7863] [Q&A]
请问一下如何配置可以实现 Nginx 部署前端的配置

加载 oss 静态文件已经实现了, 但是 类似 nginx 的 try files加载默认文件需要如何配置??

**Comment by juzhiyuan** [comment]
@88fantasy Follow this one https://apisix.apache.org/zh/docs/apisix/customize-nginx-configuration/

**Comment by badx** [comment]
route config
![9551665366493_ pic](https://user-images.githubusercontent.com/3362152/194789869-644a222e-659e-42ae-b14b-c24a82e2b9b0.jpg)
upstream config
![9561665366520_ pic](https://user-images.githubusercontent.com/3362152/194789878-127f36e1-c3ab-4959-9f99-4135b1f015cc.jpg)
vue use history mode
